examples/ct/ctmapper/ctmapperpb/ct_mapper.proto

```diff
@@ -16,6 +16,12 @@ syntax = "proto3";
 
 package ctmapperpb;
 
+message MapperMetadata {
```

Comment:
nit: Please document proto and fields.

Reply:
OK ... in fairness it wasn't documented to start with!  I'll improve on that :)

Reply:
Apologies for pushing that to you. I'm glad I did, though, as it's a good improvement.  ;)